back/src/Guide/Types/Core.hs

```diff
@@ -216,6 +232,33 @@ instance Aeson.ToJSON Item where
   toJSON = Aeson.genericToJSON Aeson.defaultOptions {
     Aeson.fieldLabelModifier = over _head toLower . drop (T.length "item") }
 
+instance Aeson.FromJSON Item where
+  parseJSON = Aeson.withObject "Item" $ \o -> do
+    itemUid         <- o Aeson..: "uid"
+    itemName        <- o Aeson..: "name"
+    itemCreated     <- o Aeson..: "created"
+    itemHackage     <- o Aeson..:? "hackage"
+    summary         <- o Aeson..: "summary"
+    itemSummary     <- toMarkdownBlock <$> summary Aeson..: "text"
```

Comment:
Again, `toMarkdownBlock` is not needed because MarkdownBlock already has a FromJSON instance.